chrome/locale/en-US/zotero/zotero.properties

```diff
@@ -111,7 +111,11 @@ attachmentBasePath.clearBasePath.button                        = Clear Base Dire
 dataDir.notFound						= The Zotero data directory could not be found.
 dataDir.previousDir					= Previous directory:
 dataDir.useProfileDir				= Use %S profile directory
+dataDir.existingDirInDropbox.title  = Data Directory In Dropbox
+dataDir.existingDirInDropbox.text   = Your current data directory appears to be in a Dropbox directory. This might lead to data corruption.\n\nChange Zotero data directory now?
 dataDir.selectDir					= Select a Zotero data directory
+dataDir.selectedDirInDropbox.title		= Directory In Dropbox
```

Comment:
If we went specific, I'd think we'd want something like "Unsafe Data Directory Location" (since we might apply this to other locations in the future), but I think we could just use general.warning ("Warning") here.